backend/src/api/mutation.rs

```diff
@@ -162,4 +164,54 @@ impl Mutation {
     async fn remove_block(id: Id, context: &Context) -> ApiResult<RemovedBlock> {
         BlockValue::remove(id, context).await
     }
+
+    /// Atomically mount a series into an (empty) realm;
```

Comment:
Is there a deeper reason why you chose to use `;` here instead of simply a `.`? Not important, I was just confused :D

Reply:
Nope. x)